### PR TITLE
Updated banner image to SVG

### PR DESCRIPTION
**Pull request by aFinchy**
## Change summary

- Updated banner to SVG
- Removed old PNG File

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Comment by aFinchy** [comment]
Capture121

File: assets/sass/custom.sass [hunk 1]
```diff
@@ -459,7 +459,7 @@ a.top-banner
       max-width: 100%
 
     .banner-logo
-      background-image: url('/images/events/o3dcon_2022_1.png')
+      background-image: url('/images/events/o3d_monochrome_con_logo.svg')
```
Comment (AMZN-daimini) [inline assets/sass/custom.sass:462]: Minor: could you add `margin-top: -2px` here? That way the logo would look centered:
image

**Review by AMZN-daimini — APPROVED** [review]
Thank you for fixing this, looks miles better on my screen :D 

image

Only recommending a minor adjustment :)

File: assets/sass/custom.sass (hunk 1 above)
Comment (aFinchy) [reply assets/sass/custom.sass:462]: Yes, Will do that now, :)

**Comment by aFinchy** [comment]
Capture1211
Capture121

**Review by FiniteStateGit — APPROVED** [review]

**Pull request merged**